alacritty/src/display.rs

```diff
@@ -206,6 +218,11 @@ impl Display {
             // actually draw something into it.
```

Comment:
The comments on this block should be updated and things should be moved closer to where the `swap_buffers` is actually performed. Since it just seems a bit confusing otherwise.

Reply:
Done.